app/queries/servers/channel.ts

```diff
@@ -37,6 +37,11 @@ export const prepareMyChannelsForTeam = async (operator: ServerDataOperator, tea
             pinned_post_count = storedInfo.pinnedPostCount;
         }
 
+        const member = memberships.find((m) => m.channel_id === c.id);
```

Comment:
Naive approach to move the last_post_at from the channel to the membership model. If we update the memberships from the server in any other different way, we would have to also consider this.

Reply:
still struggling to see the need of `lastPostAt` we should look into where we are reading this and for what use cases.